audb/core/config.py

```diff
@@ -91,6 +91,17 @@ class config:
     CACHE_ROOT = global_config['cache_root']
     r"""Default user cache folder."""
 
+    CACHED_VERSIONS_TIMEOUT = 10
+    r"""Timeout to acquire access to cached versions.
+
+    Maximum wait time to acquire access to cached versions
+    if they are currently blocked by another thread or process.
+    If timeout is reached,
+    all missing files will be downloaded from backend instead.
+    If timeout < 0 will wait until the cache folders can be accessed.
+
+    """
```

Comment:
I don't think that this is something a user should have to care about.
I would propose to simply move it to `audb/core/define.py`.

Reply:
What if my internet connection is down and copying from another version is my only option and I don't want to risk a timeout? Ok, I must admit it's not a very likely use-case, but still - who knows :)

Reply:
If your internet connection is down it is very unlikely that another person can access the cache, this means it is in your hands to avoid a timeout. Also I would still think it is very unlikely that it will occur. E.g. for the shared cache on our compute servers `audb` only failed once for me in several month due to another process accessing the same cache folder.

Reply:
Ok, maybe more likely use-case is that the backend is down. I also agree that even race conditions are not too likely the way we use `audb` at the moment, but this depends on how 'audb` is used. E.g. think about a large number of processes running in parallel requesting always few files from a large database.

But ok, if you think we should move it to `define`, let's do it. 10 seconds are good fit?

Reply:
Yes, 10 seconds seems fine.

Reply:
Changed it accordingly and updated the description.